layouts/article.html

```diff
@@ -15,6 +15,11 @@
         <div class="markdown-body document-content">
           <% section = lookup_section('toc', @item[:title]) %>
           <h1><%= section << ' ' << @item[:title] if section %></h1>
+          <h1 class="improve">
+            <a href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content" + @item.identifier %>">
```

Comment:
I think this might look nicer inlined with the `h1` above, instead of on a separate `h1`:

``` erb
<h1><%= section << ' ' << @item[:title] if section %></h1> | <a href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content" + @item.identifier %>"> <span class="octicon octicon-pencil"></span>  Improve this page  </a>
```

Also, I think this might break the current test suite for new files. You should add `data-proofer-ignore` to have this link skipped:

``` erb
<a href="<%= "https://github.com/atom/flight-manual.atom.io/edit/master/content" + @item.identifier %>" data-proofer-ignore>
```